src/General/Alert/Alert.tsx

```diff
@@ -9,58 +9,44 @@ import {
   CloseIcon,
 } from '../Icon/components';
 import {
+  AlertClose,
   AlertContainer,
-  AlertContent,
-  AlertMessage,
   AlertIcon,
-  AlertTypeIconContainer,
+  AlertMessage,
 } from './AlertStyle';
 
 import { Greyscale, PrimaryColor, SecondaryColor } from '../../Utils/Colors';
+import { Box } from '../../Layout/Box';
+import { isUndefined } from 'lodash';
+import { DelayedUnmount } from '../../Utils/DelayedUnmount';
 
 export const Alert = ({
-  isOpen,
+  isOpen: externalIsVisible,
   autoClose,
-  onClose,
+  onClose: externalOnClose,
   type = 'info',
   className,
   message,
 }: Props) => {
-  const alertContainerRef = React.useRef<HTMLDivElement>();
-  const autoCloseTimeout = React.useRef<ReturnType<typeof setTimeout>>();
-  const [isVisible, setIsVisible] = React.useState<boolean>(isOpen);
+  const [internalIsVisible, setInternalIsVisible] = React.useState<boolean>(
+    true
```

Comment:
Cannot :grin: If you do that then there won't be any way to close the alert (without controlling it).

Reply:
Base on this https://github.com/glints-dev/glints-aries/pull/616#discussion_r589909866, I think the alert should be able to be closed without controlling it.

Reply:
It can, if you don't control it then the internals take over. Behaves just the same, closes after the timeout, closes when clicked, and stays open until clicked without timeout.